src/picongpu/include/fields/laserProfiles/laserPlaneWave.hpp

```diff
@@ -46,48 +69,50 @@ namespace picongpu
             double envelope = double(AMPLITUDE );
             float3_X elong(float3_X::create(0.0));
 
-            // a NON-symmetric (starting with phase=0) pulse will be initialized at position z=0 for
-            // a time of RAMP_INIT * PULSE_LENGTH + LASER_NOFOCUS_CONSTANT = INIT_TIME.
-            // we shift the complete pulse for the half of this time to start with
-            // the front of the laser pulse.
             const double mue = 0.5 * RAMP_INIT * PULSE_LENGTH;
 
             const double w = 2.0 * PI * f;
+            const double tau = PULSE_LENGTH / sqrt( 2.0 );
```

Comment:
as @PrometheusPi realized correctly, this should have been
```c++
const double tau = PULSE_LENGTH * sqrt( 2.0 );
```

since it replaces `/ PULSE_LENGTH / sqrt( 2.0 )` in
```c++
const double exponent = (runTime - startDownramp) / tau;
```